File: cash.py
from cs50 import get_float

change = 0
while change <= 0:
    change = get_float("Change owed: ")

cents = change * 100
coins = 0

while cents > 0:
    if cents >= 25:
        cents -= 25
        coins += 1
    elif cents >= 10 and cents < 25:
        cents -= 10
        coins += 1
    elif cents >= 5 and cents < 10:
        cents -= 5
        coins += 1
    elif cents >= 0 and cents < 5:
        cents -= 1
        coins += 1

print(coins)

File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change;
    do
    {
        change = get_float("Change owed: ");
    }
    while (change < 0);

    int cents = round(change * 100);
    int coins = 0;
    do
    {
        if (cents >= 25)
        {
            cents = cents - 25;
            coins++;
        }
        else if (cents >= 10 && cents < 25)
        {
            cents = cents - 10;
            coins++;
        }
        else if (cents >= 5 && cents < 10)
        {
            cents = cents - 5;
            coins++;
        }
        else if (cents > 0 && cents < 5)
        {
            cents = cents - 1;
            coins++;
        }
    }
    while (cents > 0);

    printf("%i\n", coins);
}
